templates/mail/self-serve/app-passwords.html

```diff
@@ -0,0 +1,10 @@
+{% extends 'mail/self-serve/_base.html' %}
+{% block page_heading %}
+  <h2>Self Serve - App Passwords</h2>
+{% endblock %}
+{% block body %}
+  <div id="selfServeForm"></div>
+  <script>
+    window._page.appPasswords = {{ app_passwords|safe }};
```

Comment:
alright - I think I see! this is how you're passing the labels to the vue app :+1: